doc/api/fs.md

```diff
@@ -1082,6 +1082,8 @@ added: v22.0.0
   * `cwd` {string} current working directory. **Default:** `process.cwd()`
   * `exclude` {Function} Function to filter out files/directories. Return
     `true` to exclude the item, `false` to include it. **Default:** `undefined`.
+  * `overrides` {Object} override the default `minimatch` behavior with
```

Comment:
Opinion only, I don't think `Node.js` should expose API like this.
It means this API will always be tie to `minimatch` and will becomes a resistance on enhancement or provide `node` vendor-ed one.